tools/clang/test/SemaHLSL/derivatives-allowed-on-mesh-nodes.hlsl

```diff
@@ -0,0 +1,27 @@
+// RUN: %dxc -Tlib_6_9 %s -verify
+
+// expected-no-diagnostics
+// previously, we expected 2 diagnostics: 
+// {{overload of intrinsic CalculateLevelOfDetail with SamplerComparisonState requires shader model 6.8 or greater}}
+// {{overload of intrinsic CalculateLevelOfDetailUnclamped with SamplerComparisonState requires shader model 6.8 or greater}}
```

Comment:
I don't see why it's necessary to have these, it's just confusing.  We never actually expected these diagnostics for mesh node shaders did we?

Reply:
Not technically necessary, I thought it was extra information that could be useful. Those diagnostics I mentioned would be emitted without my logical changes. I can remove them.